nixos/hardware/.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: nix
```

Comment:
Would make sense to have this file removed, I think.

Reply:
Yeah, we can think about how to integrate it into the existing tests later.